### PR TITLE
Add support for --ignore

### PR DESCRIPTION
**Pull request by dividedmind**
[delivers #95249888]

### CODE REVIEW

**Comment by inscitiv-ops-dev** [comment]
Can one of the admins verify this patch?

**Comment by dividedmind** [comment]
There would be no way to test it then (inside go).

**Comment by dividedmind** [comment]
Also, I know it's inside the CLI package, but besides being untestable it's kind of awkward to exit from inside a library-ish function.

File: command/action.go [hunk 1]
```diff
@@ -71,20 +80,31 @@ func runAction(args []string, provider, filepath, yamlInline string, subs map[st
...
-		env = append(env, envvar)
+		if envvar.error == nil {
+			env = append(env, envvar.string)
+		} else {
+			for i := range ignores {
+				if ignores[i] == envvar.string {
+					continue EnvLoop
+				}
+			}
+			return envvar.string, envvar.error
```
Comment (kgilpin) [inline command/action.go]: Can't we print all the errors that occurred, rather than just the first one?

Comment (dustinmm80) [inline command/action.go]: I don't think we need to manually print at all, we probably should just hook io.Stderr to the command so it prints out as it executes

**Pull request merged**